requirements.txt

```diff
@@ -6,6 +6,7 @@ brax
 opencv-python
 wandb
 mediapy
+pynwb
```

Comment:
Should go in `pyproject.yaml` (rebase this branch to incorporate changes in #22 if that one isn't ready to merge to `main` yet).

Reply:
Do you mean it should *also* go in there, or that is should be moved there from here?

Reply:
Remove requirements.txt and setup.py entirely